src/createAnimatedComponent/createAnimatedComponent.tsx

```diff
@@ -251,6 +252,10 @@ export function createAnimatedComponent(
     }
 
     _getViewInfo(): ViewInfo {
+      if (this._viewInfo !== undefined) {
+        return this._viewInfo;
+      }
```

Comment:
Just to be sure - did you tested those changes with fast refresh and hot reload?

Reply:
Yep, it seems to work fine.